master/data_partition.go

```diff
@@ -700,11 +700,11 @@ func (partition *DataPartition) ToProto(c *Cluster) *proto.DataPartitionInfo {
 		Zones:                   zones,
 		MissingNodes:            partition.MissingNodes,
 		VolName:                 partition.VolName,
-		VolID:                   partition.VolID,
+	VolID:                   partition.VolID,
```

Comment:
Please note that before submitting the code, please use `goimport` or `go fmt` tool to optimize the code format.